### PR TITLE
Plangrid docs

### PR DESCRIPTION
**Pull request by bennettgo**
### Which issues does this fix?
PlanGrid Global scope. This documentation is pegged for release on Wednesday in tandem with global release of connector

### PR submission checklist
- [x] Optimised every new image
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [x] Added new files to SUMMARY.md

### CODE REVIEW

File: connectors/plangrid.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+![PlanGrid connection page on Workato](/assets/images/plangrid/workato-plangrid-connection.png)
+
+* **Connection name**
+
+  Give the PlanGrid connection a unique name that identifies which PlanGrid instance it is connected to.
+
+* **Client ID**
+
+  Create a client ID by registering an application [here](https://enterprise.plangrid.com). Navigate to your organisation and the OAuth section before creating a new OAuth App in the top right corner.
```
[the remark this answers is not in the document]
Comment (bennettgo) [reply connectors/plangrid.md]: We changed to have a set of client IDs and secrets that is specific to Workato and used by everyone making a connection

File: connectors/plangrid/object-action.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
+---
+title: Workato connectors - PlanGrid - object actions
+date: 2019-09-09 12:00:00 Z
+---
```
Comment (eeshansim) [inline connectors/plangrid/object-action.md]: for a long document like this, better to add `isTocVisible: true` to the header

File: connectors/plangrid/object-action.md [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+* [Get RFI status](#rfi)
+* [Get Sheet](#sheet)
+* [Create, Get Sheet packet](#sheet)
+* [Create, Upload Sheet version](#sheet)
+* [Get Snapshot](#snapshot)
+* [Create, Get, Update Task](#task)
+* [Create, Get, Update Task list](#task)
+* [Create, Get User](#user)
+* [Get Role](#role)
+
```
Comment (eeshansim) [inline connectors/plangrid/object-action.md]: extra line break

File: connectors/plangrid/object-trigger.md [hunk 4]
```diff
@@ -0,0 +1,397 @@
+---
+title: Workato connectors - PlanGrid - object triggers
+date: 2019-09-09 12:00:00 Z
+---
```
Comment (eeshansim) [inline connectors/plangrid/object-trigger.md]: for a long document like this, better to add `isTocVisible: true` to the header

File: connectors/plangrid/object-action.md [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+
+### Project
```
Comment (eeshansim) [inline connectors/plangrid/object-action.md]: use h2 (`##`) for object titles to make it visible in the TOC

File: connectors/plangrid.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+![PlanGrid connection page on Workato](/assets/images/plangrid/popup-dialog-page.png)
+
+* **Email**
+
+  Email of PlanGrid account.
+
+* **Password**
+
+  Password of PlanGrid account.
+
```
Comment (eeshansim) [inline connectors/plangrid.md]: extra line break

File: connectors/plangrid/object-trigger.md [hunk 7]
```diff
@@ -0,0 +1,397 @@
...
+* [New or New/updated project](#project)
+* [New or New/updated Task](#task)
+* [New or New/updated RFI](#rfi)
+* [New or New/updated Field Report](#field-report)
+* [New Sheet](#sheet)
+* [New Document](#document)
+* [New Annotation](#annotation)
+* [New Photo](#photo)
+* [New Snapshot](#snapshot)
+
```
Comment (eeshansim) [inline connectors/plangrid/object-trigger.md]: extra line break

File: connectors/plangrid.md (hunk 6 above)
Comment (bennettgo) [reply connectors/plangrid.md]: done

File: connectors/plangrid/object-action.md (hunk 5 above)
Comment (bennettgo) [reply connectors/plangrid/object-action.md]: done

File: connectors/plangrid/object-trigger.md (hunk 4 above)
Comment (bennettgo) [reply connectors/plangrid/object-trigger.md]: done

File: connectors/plangrid/object-action.md (hunk 2 above)
Comment (bennettgo) [reply connectors/plangrid/object-action.md]: done

File: connectors/plangrid/object-action.md (hunk 3 above)
Comment (bennettgo) [reply connectors/plangrid/object-action.md]: done

**Pull request merged**